1-Draft/RFCNNNN-Multi-Line-Continuance-For-Commands.md

```diff
@@ -0,0 +1,124 @@
+* ad hoc could support this syntax as well (PSReadline could wait for a double-enter when in multi-line command parsing mode)
+* no breaking changes (a standalone @ is currently an unrecognized token in PowerShell no matter where it is used).
+
+**Cons:**
+
+* if no command-terminating token is used to terminate the command, a blank line is required after the command as terminator (but, users can opt to use a semi-colon or some other command-terminating token if they don't want the blank line)
+
+## Motivation
+
+    As a script/module author,
+    I can wrap commands across multiple lines easily and intuitively without backticks or splatting
+    so that my scripts remain easy to write and maintain while still giving me the benefits of Intellisense and tab completion.
+
+## Specification
+
+* expand the command parser to accept multi-line command parameters/arguments after an at symbol (`@`) is encountered at the end of a line
+* terminate multi-line commands when the parser encounters either a blank line or a command-terminating token (a command-terminating token will be a pipe symbol, redirection operator, closing enclosure, or semi-colon when the command does not have the stop-parsing sigil in its arguments, or simply pipe symbol or redirection operator the command does have the stop-parsing sigil in its arguments)
+
+## Alternate Proposals and Considerations
+
+### A different sigil
+
+The original draft of this RFC included different options for the sigil that could be used to enter multi-line parameter/argument parsing mode, and others were presented in the discussion however none of the other sigils that were presented could be used without breaking changes. When considering an alternate sigil, it must be something that can be identified as a unique token without breaking commands such as Write-Host (which can write many sigils to the console) or commands external to PowerShell.
+
+### Enclosures instead of a sigil
+
+The original draft also included some proposals for enclosures instead of a leading sigil. While it may seem up front that enclosures make good sense, some thought needs to be given to the stop-parsing sigil, which causes PowerShell to treat closing enclosures as arguments for a command.
+
+For example, consider this syntax:
+
+```PowerShell
+. {cmd --% dir}
+```
+
+That command will not parse because there is no closing brace for the script block because what appears to be a closing brace is placed after the stop-parsing sigil. To correct this, the closing brace must be placed on a separate line, but in a multi-line command you cannot do that (because the command is multi-line, so where would the parser terminate after a stop-parsing sigil) and therefore, unless the sigil used to identify the end of the multi-line parameter/arguments was one that could be respected by the stop-parsing sigil, and safely be introduced without risk to breaking changes, enclosures simply cannot be used.
```

Comment:
This whole example is just confusing to me. It's hypothetical, and not multi-line. Wouldn't I just add two newlines instead of one, and move on in life?

Reply:
Updated with a different example. The point is to identify why closures were rejected instead of a leading sigil and two newlines.